### DISCUSSION TITLE
Anything unusual about my repo structure for `mkdocstrings`?

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [mkdocstrings/mkdocstrings#585] [Q&A] [answered]
I have been trying to follow any of the examples in the `mkdocstrings` docs only to be met with nothing rendering. I must be doing something wrong but can't figure out what and could use some insight.

![image](https://github.com/mkdocstrings/mkdocstrings/assets/8674072/96612ea6-3489-49cd-a35d-21d97a840fa1)

- Here is my repo: https://github.com/Quantum-Accelerators/quacc
- Here is my `mkdocs.yml`: https://github.com/Quantum-Accelerators/quacc/blob/main/mkdocs.yml
- Here is the broken docs page: https://quantum-accelerators.github.io/quacc/reference/calculators/

I have made sure I did the following:

- I have `mkdocstrings[python]` pip install'd
- No errors appear when doing `mkdocs serve`

However, despite the above, all that appears in the docstring is the text as-written:

Is there anything that I've set up incorrectly? I am using `mkdocstrings` 0.2.0 on Ubuntu.

**Comment by pawamoy** [comment, marked as answer]
Your `plugins` key in mkdocs.yml is under the `theme` one. De-indent it and it should be good :slightly_smiling_face:

**Reply by Andrew-S-Rosen** [reply]
Wow, it is concerning I did not notice that 😅 Thank you so much!! Now I'm excited to get some docstrings!!
